.github/workflows/ci.yml

```diff
@@ -1,21 +1,50 @@
+# Github workflow files do not support YAML anchors.
+
 name: CI
 on:
   push:
     branches:
       - master
   pull_request:
+
 jobs:
   check:
     name: Check
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v1
-    - uses: actions-rs/toolchain@v1
-      with:
-        toolchain: stable
-    - uses: actions-rs/cargo@v1
-      with:
-        command: check
+      - uses: actions/checkout@v1
+      - uses: actions-rs/toolchain@v1
+        id: toolchain
+        with:
+          profile: minimal
+          toolchain: stable
+      - name: Chown cargo registry because of caching
+        run: sudo chown -R runner ~/.cargo/registry
+      - name: Generate lockfile
+        uses: actions-rs/cargo@v1
+        with:
+          command: generate-lockfile
```

Comment:
I have already added the `Cargo.lock` file to the repository, so please remove this step.

Reply:
Done